sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -443,6 +443,76 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
       }
   }
 
+  testPartitionedTable("INSERT INTO a partitioned table, use nulls for missing columns") {
```

Comment:
This case doesn't seem to related to insertable view

Reply:
Removed this test since it's not relevant.